libs/SalesforceSDKCore/SalesforceSDKCore/Classes/RestAPI/SFRestAPI.m

```diff
@@ -278,6 +286,13 @@ - (void)replayRequestIfRequired:(NSData *)data response:(NSURLResponse *)respons
     }
 }
 
+- (void)createAndStoreLogoutEvent:(NSError *)error {
+    NSMutableDictionary *attributes = [[NSMutableDictionary alloc] init];
+    attributes[@"errorCode"] = [NSNumber numberWithInteger:error.code];
+    attributes[@"errorDescription"] = error.localizedDescription;
+    [SFSDKEventBuilderHelper createAndStoreEvent:@"userLogout" userAccount:nil className:NSStringFromClass([self class]) attributes:attributes];
```

Comment:
That makes sense.
Should I fix it [here](https://github.com/forcedotcom/SalesforceMobileSDK-iOS/blob/master/libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Network/Network/OAuth/CSFSalesforceOAuthRefresh.m#L86) too?